setup.cfg

```diff
@@ -30,6 +30,12 @@ ignore_errors = True
 
 # tox test environment for generating licenses
```

Comment:
This comment could do with being reworded. i.e. now we have tox env's for more than just generating licenses?